.github/workflows/release.yml

```diff
@@ -0,0 +1,136 @@
+name: Release and Propose an Upgrade
+
+env:
+  SUBWASM_VERSION: 0.20.0
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - 'main'
+
+jobs:
+  srtool:
+    if: ${{ github.event.pull_request.merged == true && contains(github.event.pull_request.title, 'release:') }}
```

Comment:
This job will only run if:
*  the pull request is closed and merged to main branch;
*  the pull request has the _**"release:"**_ in its title